problems/variables.js

```diff
@@ -7,6 +7,13 @@
 //  * For example, if you were born in 1988, then in 2026 you'll be either 37 or 38, depending on what month it is in 2026.
 //  * Log them to the screen like so: "I will be either `ageMin` or `ageMax` in `futureYear`", substituting the values.
 
+// console.log ("problem one: age calculator");
+// const birthYear = 1992;
+// const futureYear = 2022;
+// const ageMin = futureYear - birthYear;
+// const ageMax = ageMin+1;
```

Comment:
You have your min and maxed confused but good approach!

Max should be future - birth, min should be Max - 1